chain/consensus/filcns/upgrades.go

```diff
@@ -63,8 +63,8 @@ import (
 	"github.com/filecoin-project/lotus/node/bundle"
 )
 
-//go:embed FVMLiftoff.txt
```

Comment:
delete `FVMLiftoff.txt` while you're at it?

Reply:
done